src/org/exist/protocolhandler/embedded/EmbeddedUpload.java

```diff
@@ -102,7 +103,7 @@ public void stream(XmldbURL xmldbURL, InputStream is, Subject user) throws IOExc
      * @param tmp Document that is inserted.
      * @throws IOException
      */
-    public void stream(XmldbURL xmldbURL, File tmp) throws IOException {
+    public void stream(XmldbURL xmldbURL, Path tmp) throws IOException {
```

Comment:
Ah right, sorry I cherry-picked a commit which should not have been included. I have now removed it.